src/main/java/com/dreamteam/database/main.java

```diff
@@ -201,15 +198,14 @@ public static void runMenu() {
 	}
 
 	//	***************************************************************************
-	// TODO create a file with a date stamp like daily-report-10-18.txt under 'files/reports'
 
-	/** TODO Write the networth of all assets to a daily report file.
-	 * 
+	/**
+	 *
+	 * @param date
 	 */
-
 	public static void dailyAssetsReport(String date) {  //prints out report to console and txt file with assets in product_database and customer orders and sales in order_database
 		NumberFormat formatter = NumberFormat.getCurrencyInstance(); //to format the print statements in dollar form
-		String report_path = "files/reports/dailyreport_" + date + ".txt";
+		String report_path = "files/reports/daily-report_" + date + ".txt";
 		
 		try {
```

Comment:
This should probably be a try with resources to, but the file will need to be created via a file writer object.